assets/component-slider.css

```diff
@@ -130,7 +130,8 @@ slider-component {
 .slider-counter {
   margin: 0 1.2rem;
   display: flex;
-  min-width: 2rem;
+  justify-content: center;
+  width: 2rem;
```

Comment:
What about making the `min-width` value a bit higher instead of setting a fix value 🤔  Like `min-width: 3rem;`

Reply:
Nice 👌